src/cairo.mk

```diff
@@ -3,11 +3,11 @@
 PKG             := cairo
 $(PKG)_WEBSITE  := http://cairographics.org/
 $(PKG)_IGNORE   :=
-$(PKG)_VERSION  := 1.14.6
-$(PKG)_CHECKSUM := 613cb38447b76a93ff7235e17acd55a78b52ea84a9df128c3f2257f8eaa7b252
+$(PKG)_VERSION  := 1.15.4
```

Comment:
Using the snapshot seems fine, from the release notes:
```
This new snapshot incorporates changes over the past year since the
1.15.2 snapshot, including all the fixes from the 1.14 release series.
```